vignettes/modification.Rmd

```diff
@@ -119,6 +126,29 @@ xml_add_child(t1, read_xml("<test/>"))
 x
 ```
 
+## Removing nodes ##
+The `xml_remove_node()` can be used to remove a node (and it's children) from a
+tree. The default behavior is to unlink the node from the tree, but does not
+free the memory for the node, so R objects pointing to the node are still
+valid. This allows code like the following to work without crashing R
```

Comment:
I think you need to be a bit clearer about leaking memory here: "This means the default behaviour of `xml_remove()` is to leak memory, but it allows code like the following...`